examples/pipelines/starcoder/pipeline.py

```diff
@@ -46,7 +48,20 @@
 pipeline.add_op(load_from_hub_op)
 pipeline.add_op(filter_line_length_op, dependencies=load_from_hub_op)
 pipeline.add_op(filter_comments_op, dependencies=filter_line_length_op)
-pipeline.add_op(pii_redaction_op, dependencies=load_from_hub_op)
+pipeline.add_op(pii_redaction_op, dependencies=filter_comments_op)
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--local", action="store_true")
+    if parser.parse_args().local:
+        compiler = DockerCompiler()
+        extra_volumes = [
+            "$HOME/.config/gcloud/application_default_credentials.json:/root/.config/gcloud/application_default_credentials.json:ro"
+        ]
+        compiler.compile(pipeline=pipeline, extra_volumes=extra_volumes)
+        logger.info("Run `docker-compose up` to run the pipeline.")
```

Comment:
```suggestion
        logger.info("Run `docker compose up` to run the pipeline.")
```

Reply:
oh it seems like it works but I had to install it. never mind you can keep it as is

Reply:
It doesn't for me. `docker compose` is the recommended way ([link](https://stackoverflow.com/a/66516826/4098821))